Lib/_collections_abc.py

```diff
@@ -430,25 +430,13 @@ def __new__(cls, origin, args):
             raise TypeError(
                 "Callable must be used as Callable[[arg, ...], result].")
         t_args, t_result = args
-        if isinstance(t_args, list):
+        if isinstance(t_args, (tuple, list)):
             args = (*t_args, t_result)
         elif not _is_param_expr(t_args):
             raise TypeError(f"Expected a list of types, an ellipsis, "
                             f"ParamSpec, or Concatenate. Got {t_args}")
         return super().__new__(cls, origin, args)
 
-    @property
```

Comment:
I think we have to keep `__parameters__`, it's de facto a public API by now.

Reply:
It is inherited from GenericAlias now.